zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
Then how about adjust build order? Move 'zeppelin-web' module to the front of build order, so it can fail early?

We never know user want to (or allowed to) install any package on the system or not.